wgpu-native/src/command/transfer.rs

```diff
@@ -160,9 +160,10 @@ pub extern "C" fn wgpu_command_buffer_copy_buffer_to_texture(
         image_extent: conv::map_extent(copy_size),
     };
     let cmb_raw = cmb.raw.last_mut().unwrap();
+    let stages = all_image_stages() | all_image_stages();
```

Comment:
This seems like a typo here and below, is it supposed to `all_buffer_stages() | all_image_stages()`?

I also meant to ask about the previous usage, could you clarify the previous `all_buffer_stages() .. all_image_stages()` and why it wasn't correct?